interface/src/app/tribe_client/geneset-search-bar.tpl.html

```diff
@@ -1,12 +1,11 @@
 <div>
   <form class="form-inline">
     <label>Organism:</label>
-    <select ng-disabled="true">
-      <option ng-selected="true" value="organism">{{organism}}</option>  
-    </select>
+    <i>{{$ctrl.organism}}</i>&nbsp;
+    <input ng-model="$ctrl.search.query" placeholder="Search text">
```

Comment:
Yeah - so this is definitely more related to how Tribe handles gene set searches. You can only really search by the gene set title or any word contained in the abstract right now, not by any genes present in the actual gene sets. I created an issue for it in the Tribe repo. in bitbucket.